src/app/nav-sidebar/nav-sidebar.component.html

```diff
@@ -0,0 +1,38 @@
+<mat-toolbar color="primary">
+  <button
+    type="button"
+    aria-label="Toggle sidenav"
+    mat-icon-button
+    (click)="drawer.toggle()"
+    *ngIf="isHandset$ | async">
+    <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
+  </button>
+  <span>ERP Admin Portal</span>
```

Comment:
What about i18n? Shouldn't we start from now with it?

Reply:
Good point. I'll create a separate issue for this.